src/test/java/org/jenkinsci/plugins/ownership/security/rolestrategy/OwnershipBasedSecurityTestHelper.java

```diff
@@ -0,0 +1,218 @@
+        ownerPermissions.addAll(Item.PERMISSIONS.getPermissions());
+        ownerPermissions.addAll(Run.PERMISSIONS.getPermissions());
+        Role ownerRole = createRole("@OwnerNoSid", ".*", ownerPermissions);
+        
+        Set<Permission> coownerPermissions = new HashSet<Permission>();
+        coownerPermissions.addAll(Item.PERMISSIONS.getPermissions());
+        coownerPermissions.addAll(Run.PERMISSIONS.getPermissions());
+        coownerPermissions.remove(Item.DELETE);
+        coownerPermissions.remove(Run.DELETE);
+        Role coOwnerRole = createRole("@CoOwnerNoSid", ".*", coownerPermissions);
+        
+        return createRoleMapForSid("authenticated", ownerRole, coOwnerRole);
+    }
+    
+    private static RoleMap getComputerRoleMap() {
+        Set<Permission> ownerPermissions = new HashSet<Permission>();
+        ownerPermissions.add(OwnershipPlugin.MANAGE_SLAVES_OWNERSHIP);
+        ownerPermissions.addAll(Computer.PERMISSIONS.getPermissions());
+        Role ownerRole = createRole("@OwnerNoSid", ".*", ownerPermissions);
+        
+        Set<Permission> coownerPermissions = new HashSet<Permission>();
+        coownerPermissions.addAll(Computer.PERMISSIONS.getPermissions());
+        coownerPermissions.remove(Computer.DELETE);
+        coownerPermissions.remove(Computer.CONFIGURE);
+        Role coOwnerRole = createRole("@CoOwnerNoSid", ".*", coownerPermissions);
+        
+        return createRoleMapForSid("authenticated", ownerRole, coOwnerRole);
+    }
+    
+    private static Role createRole(String name, String pattern, Permission ... permissions) {
+        Set<Permission> permSet = new HashSet<Permission>();
+        for (Permission p : permissions) {
+            permSet.add(p);
+        }
+        return createRole(name, pattern, permSet);
+    }
+    
+    private static RoleMap createRoleMapForSid(String sid, Role ... roles) {
+        final SortedMap<Role,Set<String>> grantedRoles = new TreeMap<Role, Set<String>>();
+        for (Role role : roles) {
+            grantedRoles.put(role, singleSid(sid));
+        }
+        return createRoleMap(grantedRoles);
+    }
+    
+    private static Set<String> singleSid(String sid) {
+        final Set<String> sids = new TreeSet<String>();
+        sids.add(sid);
+        return sids;
+    }
+    
+    // TODO: Methods below should be burned with fire when RoleStrategy gets better API
+    private static Role createRole(String name, String pattern, Set<Permission> permissions) {
+        try {
+            Constructor<Role> constructor = locateConstructor(Role.class, String.class, String.class, Set.class);
+            try {
+                constructor.setAccessible(true);
+                return constructor.newInstance(name, pattern, permissions);
+            } finally {
+                constructor.setAccessible(false);
+            }
+        } catch (Exception ex) {
+            throw new AssertionError("Cannot create role", ex);
```

Comment:
use `fail` method instead of throwing AssertionErr it by yourself. But i think it should throw RuntimeExc, as this exc lead to breaking the test, and not fail (or u should move check logic inside of test)